nipype/interfaces/ants/preprocess.py

```diff
@@ -0,0 +1,358 @@
+                        desc="This is the 4d image to be motion corrected")
+    metric_weight = traits.Float(1.0, requires=['metric_type'])
+    radius_or_bins = traits.Int(desc="", requires=['metric_type'])
+    sampling_strategy = traits.Enum("None", "Regular", "Random", None,
+                                    requires=['metric_type'])
+    sampling_percentage = traits.Either(traits.Range(low=0.0, high=1.0), None,
+                                        requires=['metric_type'])
+
+    transformation_model = traits.Enum("Affine", "Rigid", argstr="%s")
+    gradient_step_length = traits.Float(requires=['transformation_model'],
+                                        desc='')
+
+    iterations = traits.List(traits.Int, argstr='-i %s', sep='x',
+                             desc="Specify the number of iterations at each level.")
+
+    smoothing_sigmas = traits.List(traits.Float, argstr='-s %s', sep='x',
+                                   desc="Specify the amount of smoothing at each level.")
+
+    shrink_factors = traits.List(
+        traits.Int,
+        argstr='-f %s',
+        sep='x',
+        desc=("Specify the shrink factor for the virtual domain (typically "
+              "the fixed image) at each level.")
+    )
+
+    n_images = traits.Int(
+        argstr="-n %d",
+        desc=("This option sets the number of images to use to construct the "
+              "template image.")
+    )
+
+    use_fixed_reference_image = traits.Bool(
+        True,
+        argstr="-u %d",
+        desc=("use a fixed reference image instead of the neighor in the time "
+              "series.")
+    )
+
+    use_scales_estimator = traits.Bool(
+        True,
+        argstr="-e %d",
+        desc="use the scale estimator to control optimization."
+    )
+
+    use_estimate_learning_rate_once = traits.Bool(
+        False,
+        argstr="-l %d",
+        desc=("turn on the option that lets you estimate the learning rate "
+              "step size only at the beginning of each level. Useful as a "
+              "second stage of fine-scale registration.")
+    )
+
+    write_displacement = traits.Bool(
+        False,
+        argstr="-w %d",
+        desc="Write the low-dimensional 3D transforms to a 4D displacement field"
+    )
+
+
+
+class AntsMotionCorrOutputSpec(TraitedSpec):
+    '''Output spec for the antsMotionCorr command'''
+    average_image = File(exists=True, desc="Average of an image")
+    composite_transform = File(desc="Composite transform file")
+    inverse_composite_transform = File(desc="Inverse composite transform file")
+    warped_image = File(desc="Outputs warped image")
+    inverse_warped_image = File(desc="Outputs the inverse of the warped image")
+    save_state = File(desc="The saved registration state to be restored")
+    displacement_field = File(desc=("4D displacement field that captures the "
+                                    "affine induced motion at each voxel"))
+
+class AntsMotionCorr(ANTSCommand):
```

Comment:
Since this will live under `nipype.interfaces.ants` I'd say the "Ants" prefix is unnecessary. What about `MotionCorrection` instead of `AntsMotionCorr`? (similarly to `antsRegistration` which is just `Registration` in nipype).